.github/workflows/main.yaml

```diff
@@ -6,11 +6,13 @@ jobs:
     strategy:
       matrix:
         libslirp_commit: [master, v4.3.1, v4.2.0, v4.1.0]
```

Comment:
Can we rather use Ubuntu 20.04 instance of GitHub Action so that we can use real Ubuntu 20.04 kernel

Reply:
Makes sense to test that too, right. (Although the issue from gh-232 should be just userspace, right?)
My intent with this PR was just to show that it fails without gh-232 and works if it's applied. Hence, it's rather temporary and in a non-merge-able state.
However, the tests fail at another check, currently:
```
+ slirp_pid=2071
+ wait_for_network_device 2045 tun11
+ COUNTER=0
+ '[' 0 -lt 40 ']'
+ nsenter --preserve-credentials -U -n --target=2045 ip addr show tun11
+ slirp4netns -c 2045 --cidr 10.0.135.128/25 tun11
Device "tun11" does not exist.
+ sleep 0.5
sent tapfd=5 for tun11
received tapfd=5
+ let COUNTER=COUNTER+1
+ '[' 1 -lt 40 ']'
+ nsenter --preserve-credentials -U -n --target=2045 ip addr show tun11
2: tun11: <BROADCAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UNKNOWN group default qlen 1000
    link/ether 9e:69:47:51:c9:3c brd ff:ff:ff:ff:ff:ff
    inet 10.0.135.228/25 brd 10.0.135.255 scope global tun11
       valid_lft forever preferred_lft forever
    inet6 fe80::9c69:47ff:fe51:c93c/64 scope link tentative 
       valid_lft forever preferred_lft forever
+ break
+ trap cleanup EXIT
++ nsenter --preserve-credentials -U -n --target=2045 ip -json a show dev tun11
++ jq -r '.[1].addr_info[0].local'
+ ip=null
+ [[ null = 10.0.135.228 ]]
+ cleanup
+ kill -9 2045 2071
FAIL tests/test-slirp4netns-cidr.sh (exit status: 1)
```